### PR TITLE
Update enable-the-report-message-add-in.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected the navigation to deploy add-in as it has changed in the new admin center.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3148

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

File: microsoft-365/security/office-365-security/enable-the-report-message-add-in.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ To learn how to use the add-in, see [Use the Report Message add-in](https://supp
 > [!NOTE]
 > It could take up to 12 hours for the add-in to appear in your organization.
 
-1. In the Microsoft 365 admin center, go to the **Services & add-ins** page at <https://admin.microsoft.com/AdminPortal/Home#/Settings/ServicesAndAddIns>, and then click **Deploy Add-In**.
+1. In the Microsoft 365 admin center, go to the **Settings**, **integrated Apps** & **Add-ins** page at https://admin.microsoft.com/AdminPortal/Home#/Settings/AddIns, and then click **Deploy Add-In**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/enable-the-report-message-add-in.md]:
```suggestion
1. In the Microsoft 365 admin center, go to the **Settings, integrated Apps & Add-ins** page at https://admin.microsoft.com/AdminPortal/Home#/Settings/AddIns, and then click **Deploy Add-In**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrisda Please merge this PR. Thanks.

**Pull request merged**